graphics/opencollada/Portfile

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        KhronosGroup OpenCOLLADA 1.6.68 v
+name                opencollada
+categories          graphics
+platforms           darwin
+license             MIT
+maintainers         @jasonliu-- openmaintainer
+homepage            http://www.opencollada.org
+description         stream-based library for reading and writing \
+                    COLLADA files
+long_description    OpenCOLLADA is a library that allows computer \
+                    graphics software applications to read and write \
+                    COLLADA (COLLAborative Design Activity) files. \
+                    COLLADA files are typically XML-based documents \
+                    identified with a .dae (digital asset exchange) \
+                    filename extension.
+
+checksums           rmd160  afc58cdd5ff73f8f11ae4f2da87d43817df9974d \
+                    sha256  70af2194a3a411ccc05102daa3faaa5f1ea1758120171f7f635a0e488985b2ad \
+                    size    12098976
+
+depends_build-append \
+                    port:pkgconfig
+depends_lib         port:libxml2
```

Comment:
please use `depends_lib-append`

Reply:
Why does it need to be `append`? It's only dependent on that one library.

Reply:
its good practise to always append, as it always does the right thing. Bear in mind PortGroups can (and many do) add dependencies. If you don't then append in the port, you wipe those deps. out. We quite regularly see bugs because of this.

appending *always* does the right thing, for both cases where there are no previous deps, and when there are some, so its a good habit to get into. e.g. your port will be safe if in the future a PG is added that does add a dep., or if one of the ones you are currently using adds one.